db/data.py

```diff
@@ -460,3 +462,42 @@ def get_summary_data(mbid, offset=0):
         pass
 
     return summary
+
+def find_md5_duplicates(md5):
+    """Check if data comes from an audio file with the same md5 (Uses data->md5_encoded)
+        
+    Args: 
+        md5: The md5_encoded from the low-level data to check
+    
+    Returns:
+        The retrieved mbid if the md5 already exists
+        None if the md5 is not found
+    """
+
+    query = text("""
+                 SELECT lowlevel.mbid
+                 FROM lowlevel, lowlevel_json
```

Comment:
in other places in the server we use a JOIN, instead of a FROM/WHERE. It means exactly the same, but for consistency we should always write it with the JOIN
```
SELECT lowlevel.mbid
  FROM lowlevel
  JOIN lowlevel_json USING (id)
 WHERE lowlevel_json.data #> '{metadata,audio_properties,md5_encoded}' ? :md5
```